src/Troubleshooting/index.md

```diff
@@ -6,10 +6,7 @@ If there are any undocumented issues, feel free to [create an issue](https://git
 
 ## Checking AVX Compatibility
 
-There are 2 ways to check if your CPU supports AVX:
-
-- Open up a terminal and type the following command: `grep avx /proc/cpuinfo` it will highlight with red the AVX flags, if you get no output it means that your current CPU doesn't support AVX.
-- While having Vinegar installed type the following command: `vinegar sysinfo` or if using flatpak: `flatpak run org.vinegarhq.Vinegar sysinfo` it will show information about your system, this includes support for AVX.
+Open up a terminal and type the following command: `grep avx /proc/cpuinfo` it will highlight with red the AVX flags, if you get no output it means that your current CPU doesn't support AVX.
```

Comment:
Sysinfo is still a feature of Vinegar, under About > Troubleshooting.

AVX is no longer required to be documented since its Player-specific.